Sources/VLCLibraryViewController.m

```diff
@@ -414,10 +417,14 @@ - (void)_displayEmptyLibraryViewIfNeeded
         [self.view addSubview:self.emptyLibraryView];
         self.navigationItem.rightBarButtonItems = nil;
     } else {
-        self.navigationItem.rightBarButtonItems = @[self.editButtonItem, _rendererBarButton];
+
+        UIBarButtonItem *fixedSpace = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemFixedSpace target:nil action:nil];
+        fixedSpace.width = NAVBAR_FIXED_SPACE;
+        self.navigationItem.rightBarButtonItems = @[self.editButtonItem, fixedSpace, _rendererBarButton];
         if (UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPad) {
+            fixedSpace.width = NAVBAR_FIXED_SPACE_IPAD;
             self.displayModeBarButtonItem = [[UIBarButtonItem alloc] initWithImage:[UIImage imageNamed:@"tableViewIcon"] style:UIBarButtonItemStylePlain target:self action:@selector(toggleDisplayedView:)];
-            self.navigationItem.rightBarButtonItems = @[self.displayModeBarButtonItem, self.editButtonItem, _rendererBarButton];
+            self.navigationItem.rightBarButtonItems = @[self.displayModeBarButtonItem, self.editButtonItem, fixedSpace, _rendererBarButton];
```

Comment:
how about using a stackview here btw ?

Reply:
Might be missing something but I don't think I can assign a stackView to an array here?

Reply:
you can put a stackview in an array though

Reply:
I don’t see why.
Doesn’t it get handled by a stackview internally anyway?

Reply:
I dislike the fixedspace. I'd rather a bigger button and a stackview would take care of that by scaling them all to the same size. That would increase the hittable area